### ISSUE TITLE
Lage førsteutkast til registreringssidene (react?)

### ISSUE THREAD

**Issue by nicolhag** [fui/Bokskap-2.0#7]
Se skissene til Preben under Drive på FUI --> Bokskap --> Bokskap 2.0 --> FORSLAG 3 (BOOKING) for hvordan dette skal se ut.

**Comment by PrebenAas** [comment]
Yep. Tanken er at området med bokser er dynamisk, mao. om man scroller i det området kommer det flere bokser. Hvis man søker (begynner f.eks. på 3) kommer det _instantly_ opp bare bokser/skap i 3. etg. Og slik kan da bruker "snevre" inn skapet sitt ved hjelp av søk.
